README.md

```diff
@@ -31,4 +31,4 @@ if (user.isAdmin) {
 }
 ```
 
-For more imformation check [nextcloud.github.io/nextcloud-auth](https://nextcloud.github.io/nextcloud-auth/index.html)
+For more imformation check [nextcloud-libraries.github.io/nextcloud-auth](https://nextcloud-libraries.github.io/nextcloud-auth/index.html)
```

Comment:
This is a link back to the same page, som maybe it should just be removed? Also there is a typo: imformation instead of information,

Reply:
The link is helpful if you look at https://github.com/nextcloud-libraries/nextcloud-auth?tab=readme-ov-file